Rakefile

```diff
@@ -6,3 +6,8 @@ require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new(:spec) do |spec|
   spec.rspec_opts = '-Ispec'
 end
+
+require 'coveralls/rake/task'
+Coveralls::RakeTask.new
+task :spec_with_coveralls => [:spec, 'coveralls:push']
+
```

Comment:
> Not that it matters, but an extra line here.

Nice catch! :)

I removed the line

paresh brahmacharimayum wrote:
> In Rakefile
> <https://github.com/rprt/rspec-page-regression/pull/20#discussion_r40785466>:
>
>>  @@ -6,3 +6,8 @@ require'rspec/core/rake_task'
>>   RSpec::Core::RakeTask.new(:spec) do |spec|
>>     spec.rspec_opts ='-Ispec'
>>   end
>>  +
>>  +require'coveralls/rake/task'
>>  +Coveralls::RakeTask.new
>>  +task :spec_with_coveralls =>  [:spec,'coveralls:push']
>>  +
>
> Not that it matters, but an extra line here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rprt/rspec-page-regression/pull/20/files#r40785466>.
>